docs/automobile/scenario-creation-tutorial.md

```diff
@@ -50,28 +66,78 @@ Finally, you may add a [Fog](../reference/fog.md) node to make your world look n
 
 On Linux, the path to the SUMO binaries and to the Webots libraries directories should be added in your LD\_LIBRARY\_PATH environment variable in order to run the SUMO binaries:
 
+%tab-component "os"
+
+%tab "Linux"
+
 ```sh
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$WEBOTS_HOME/projects/default/resources/sumo/bin:$WEBOTS_HOME/lib
 ```
 
+%tab-end
+
+%tab "macOS"
+
+```sh
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$WEBOTS_HOME/Contents/projects/default/resources/sumo/bin:$WEBOTS_HOME/Contents/lib
```

Comment:
Are you sure this is useful? macOS use `DYLD_LIBRARY_PATH` instead of `LD_LIBRARY_PATH` to find shared libraries.

Reply:
I was not aware of this, so it should be as following: `export LD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:$WEBOTS_HOME/Contents/projects/default/resources/sumo/bin:$WEBOTS_HOME/Contents/lib`?

Reply:
No, it should be:
```
export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:$WEBOTS_HOME/Contents/projects/default/resources/sumo/bin:$WEBOTS_HOME/Contents/lib
```
But I am surprised that the version you documented worked out. Maybe it is simply not useful to make this export?

Reply:
My bad, that was a typo! I will fix it.